### DISCUSSION TITLE
Addition of “re-download” function

### DISCUSSION THREAD

**Discussion by teacat923** [PrismLauncher/PrismLauncher#2919] [Ideas]
If a mod has modrinth or CurseForge stuff in it, delete all the mods once and re-download a new, identical one.
This will reset the buggy mods.
It would be nice to have an option to reset the config as well.

**Comment by blryface** [comment]
... what

**Reply by teacat923** [reply]
For example, if you download a MODB that is incompatible with MODA, and you rush to disable the MODB, the entire boot configuration may become unusable because the MODA is already corrupted.
If you can identify the MODA, it is a quick story, but it is painstaking to identify and re-download specifically which MODA was incompatible with the MODB out of hundreds of mods.
Wouldn't replacing the mods with new ones in one go cut down on that effort?

**Reply by Ari-43** [reply]
> For example, if you download a MODB that is incompatible with MODA, and you rush to disable the MODB, the entire boot configuration may become unusable because the MODA is already corrupted.
> 
> If you can identify the MODA, it is a quick story, but it is painstaking to identify and re-download specifically which MODA was incompatible with the MODB out of hundreds of mods.
> 
> Wouldn't replacing the mods with new ones in one go cut down on that effort?

The mod jar files shouldn't be able to 'get corrupted' by being run (or not run) with another mod. When you disable a mod, the game just launches without it. Only the file name of the disabled mod is changed (to no longer end with .jar), and none of the other mods are modified in any way.

**Reply by blryface** [reply]
i forgot about this lol
yea they seem to not understand how mods work
